wire-ios-data-model/Source/MLS/MLSService.swift

```
@@ -740,7 +738,7 @@ public final class MLSService: MLSServiceInterface {
     }
 
     public func conversationExists(groupID: MLSGroupID) -> Bool {
```

Should this method throw as well?

---

Yes, we should let it throw as well

---

Added another TODO